app/components/Table/Filters/SearchableDropdownFilter.tsx

```diff
@@ -67,7 +67,6 @@ export default class SearchableDropdownFilter extends TableFilter<string> {
                   ...params.InputProps,
                   disableUnderline: true,
                   style: {
-                    fontFamily: "BC Sans",
```

Comment:
Could you kindly provide some context on why this was removed?

Reply:
The inclusion of this `fontFamily` property caused inconsistencies in the typeface used for the filters - part of the ticket's requirement was to have the affected filter fields' placeholder text consistent with the rest of the filter fields.

Reply:
Oh sorry, I missed that part of the ticket's context. Thank you for the clarification.

Reply:
All good! Thanks for your quick review :)